include/seqan3/range/container/aligned_allocator.hpp

```diff
@@ -28,6 +28,12 @@ namespace seqan3
  *
  * \details
  *
+ * This class allocates memory at the given alignment boundary. If the specified `alignment` is not supported by the
```

Comment:
can you always use the term `memory-alignment` instead of just `alignment` like in the documentation below? It helps to know up front that this has nothing to do with our pairwise alignment

Reply:
I will add a section about what memory alignment is.